step-issuer/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Default values for step-issuer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+deployment:
+  image:
+    repository: smallstep/step-issuer
+    pullPolicy: IfNotPresent
+    # custom add from deployments
+    tag: 0.2.0
+  resources:
+    limits:
+      cpu: 100m
+      memory: 50Mi
+    requests:
+      cpu: 100m
+      memory: 30Mi
+  terminationGracePeriodSeconds: 10
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+
+
+service:
+  type: ClusterIP
+
+# custom added from service step-issuer
+  port: 8443
+  targetPorts: https
+  controlPlane: controller-manager
+  scrape: true
+
+ingress:
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths: []
+  tls: []
```

Comment:
I think we should get rid of this, the templates do not use it, and I'm cannot think why an ingress would be useful.